src/components/navigation/Menu.tsx

```diff
@@ -5,46 +5,44 @@ import { NavHashLink } from "react-router-hash-link";
 
 // TODO: Find a solution for the behavior of contacts buttons when the checkout page is active.
```

Comment:
If this TODO is still relevant?

Reply:
- [ ] Done [873e8a0](https://github.com/AnastasiSo/Portfolio-Project/pull/27/commits/873e8a0fe0fa5738728da19d289e939fb9e09069).